media-source-respec.html

```diff
@@ -30,15 +30,17 @@
       prevRecURI: "https://www.w3.org/TR/2016/REC-media-source-20161117/",
 
       editors:  [
-      { name: "Matthew Wolenetz",  mailto:"wolenetz@google.com", url: "",
```

Comment:
If there is an option for me to retain editorship (in addition to, of course, Jean-Yves being added as an editor), that would be my preference. In that case, @chrisn, how would organization be represented for me? (e.g., "W3C Media Working Group Invited Expert" with empty companyURL?) Tt would also be good to include me also in the formerEditors as shown in this change, to indicate the period of time where I was editing on behalf of Google.) What do you think? (And thank you for pinging me about this, too, @chrisn !).

Reply:
The W3C [style guide](https://w3c.github.io/manual-of-style/#affs) gives some examples, so I would suggest following those, and [this advice](https://github.com/w3c/respec/issues/2561#issuecomment-548184120), i.e:

```
{ name: "Matthew Wolenetz",  mailto: "", url: "", company: "Invited Expert",
  note: "and before at Google Inc.", w3cid: "76912" }
```

which renders as:

> Matthew Wolenetz (Invited Expert) (and before at Google Inc.)

You can include your contact email and URL, or omit these as you prefer. As you're still an editor, there's no need to move you to `formerEditors`.

Reply:
We're happy to have Matt continue as an IE editor if he'd like!